stem/water_processes.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from abc import ABC
+
+
+@dataclass
+class WaterProcessParametersABC(ABC):
+    """
+    Abstract class which contains the parameters for a water process.
+    """
+
+
+@dataclass
+class UniformWaterPressure(WaterProcessParametersABC):
+    """
+    Class which contains the parameters for a uniform water pressure process.
+
+    Inheritance:
+        - WaterProcessParametersABC (ABC): Abstract base class which contains the parameters for a water process.
+
+    Attributes:
+        - water_pressure (float): The water pressure.
+        - is_fixed (bool): Whether the water pressure is fixed or not (default: True).
```

Comment:
is fixed you mean is constant?

Reply:
it cannot be changed during calculation, thus no water pressures are calculated during the kratos calculation